### ISSUE TITLE
Many to Many field is null in entry object

### ISSUE THREAD

**Issue by KatieMFritz** [page-8/craft-manytomany#30]
I'm perplexed!

I have a many to many relationship where one side is an Entries Subset field and the other is Many to Many. The relationship is between two entries in the same structure (different entry types).

Everything works perfectly in the CP. When templating, however, I can access the entries subset field, but the Many to Many field returns null. 

## Setup

- Craft Pro 3.2.8
- Postgres
- Many to Many - linking to master branch of this repo
- [Entries Subset](https://plugins.craftcms.com/entriessubset) 1.2.1

**Comment by KatieMFritz** [comment]
When I dump the entire entry that has the "rooms" Many to Many field, here's part of what I get:
```
["rooms"]=>
    object(Page8\ManyToMany\fields\ManyToManyField)#1574 (28) {
      ["source"]=>
      array(2) {
        ["value"]=>
        string(1) "3"
        ["type"]=>
        string(7) "section"
      }
      ["singleField"]=>
      string(2) "24"
      ["_isFresh":"craft\base\Field":private]=>
      NULL
      ["_errors":"yii\base\Model":private]=>
      NULL
      ["_validators":"yii\base\Model":private]=>
      NULL
      ["_scenario":"yii\base\Model":private]=>
      string(7) "default"
      ["_events":"yii\base\Component":private]=>
      array(0) {
      }
      ["_eventWildcards":"yii\base\Component":private]=>
      array(0) {
      }
      ["_behaviors":"yii\base\Component":private]=>
      array(0) {
      }
      ["id"]=>
      int(55)
      ["dateCreated"]=>
      object(DateTime)#1576 (3) {
        ["date"]=>
        string(26) "2019-03-05 05:05:57.000000"
        ["timezone_type"]=>
        int(3)
        ["timezone"]=>
        string(16) "America/New_York"
      }
      ["dateUpdated"]=>
      object(DateTime)#1575 (3) {
        ["date"]=>
        string(26) "2019-05-12 09:47:00.000000"
        ["timezone_type"]=>
        int(3)
        ["timezone"]=>
        string(16) "America/New_York"
      }
      ["groupId"]=>
      int(10)
      ["name"]=>
      string(5) "Rooms"
      ["handle"]=>
      string(5) "rooms"
      ["context"]=>
      string(6) "global"
      ["instructions"]=>
      string(53) "Which rooms or buildings are part of this main venue?"
      ["searchable"]=>
      bool(false)
      ["translationMethod"]=>
      string(4) "none"
      ["translationKeyFormat"]=>
      NULL
      ["oldHandle"]=>
      NULL
      ["oldSettings"]=>
      NULL
      ["columnPrefix"]=>
      NULL
      ["uid"]=>
      string(36) "2bfc99da-2314-4290-9f0d-60e3dac99efc"
      ["layoutId"]=>
      int(5)
      ["tabId"]=>
      int(519)
      ["required"]=>
      bool(false)
      ["sortOrder"]=>
      int(1)
    }
  }
  ["_fieldParamNamePrefix":"craft\base\Element":private]=>
  NULL
  ["_normalizedFieldValues":"craft\base\Element":private]=>
  array(2) {
    ["descriptionTeaser"]=>
    bool(true)
    ["rooms"]=>
    bool(true)
  }
  ["_nextElement":"craft\base\Element":private]=>
  bool(false)
  ["_prevElement":"craft\base\Element":private]=>
  bool(false)
  ["_parent":"craft\base\Element":private]=>
  NULL
  ["_prevSibling":"craft\base\Element":private]=>
  NULL
  ["_nextSibling":"craft\base\Element":private]=>
  NULL
  ["_eagerLoadedElements":"craft\base\Element":private]=>
  NULL
  ["_currentRevision":"craft\base\Element":private]=>
  NULL
  ["_errors":"yii\base\Model":private]=>
  NULL
  ["_validators":"yii\base\Model":private]=>
  NULL
  ["_scenario":"yii\base\Model":private]=>
  string(7) "default"
  ["_events":"yii\base\Component":private]=>
  array(0) {
  }
  ["_eventWildcards":"yii\base\Component":private]=>
  array(0) {
  }
```

**Comment by KatieMFritz** [comment]
Further down I see all my fields. Abridged:

```
 ["_behaviors":"yii\base\Component":private]=>
  array(1) {
    ["customFields"]=>
    object(craft\behaviors\ContentBehavior)#1505 (156) {
      ["website"]=>
        string(30) "https://www.princetonclub.com/"
      ["rooms"]=>
        NULL
...
```
Website looks correct. Rooms shouldn't be null.

**Comment by matthisamoto** [comment]
Also seeing this behavior on the Many-to-Many field. Related entries are visible in the admin page, but the variable appears as null in the template.

**Comment by engram-design** [comment]
That would be because this field never implemented any sort of value, instead rolling its own thing.

Fixed in [2.0.0](https://github.com/verbb/many-to-many/releases/tag/2.0.0)
